src/cutscenes/intro.inc.h

```diff
@@ -4,23 +4,23 @@
 
 .phases = (CutscenePhase[]) {
 	{ "cutscenes/locations/hakurei", {
-		T_NARRATOR("— The Hakurei Shrine\n"),
-		T_NARRATOR("The shrine at the border of fantasy and reality.\n"),
+		T_NARRATOR("— The Hakurei Shrine"),
+		T_NARRATOR("The shrine at the border of fantasy and reality."),
```

Comment:
Much of the removed `\n`s were placed intentionally for "paragraph" spacing, but I'd say we don't need to fix that right now. I'll be happy to merge this version now and revise the formatting and possibly scene choices in a later pass.